packages/teams-js/src/internal/communication.ts

```diff
@@ -47,15 +53,18 @@ class CommunicationPrivate {
   public static topMessageQueue: MessageRequest[] = [];
   public static nextMessageId = 0;
   public static callbacks: {
-    [id: number]: Function; // (arg1, arg2, ...etc) => void
+    [id: MessageID]: Function; // (arg1, arg2, ...etc) => void
   } = {};
   public static promiseCallbacks: {
-    [id: number]: Function; // (args[]) => void
+    [id: MessageID]: Function; // (args[]) => void
   } = {};
   public static portCallbacks: {
-    [id: number]: (port?: MessagePort, args?: unknown[]) => void;
+    [id: MessageID]: (port?: MessagePort, args?: unknown[]) => void;
   } = {};
   public static messageListener: Function;
+  public static legacyMessageIdsToUuidMap: {
+    [legacyId: number]: MessageUUID;
+  } = {};
```

Comment:
It looks like this structure is used to map messages that have ids as numbers ("legacy id") to the new UUID. Based on that, I would expect the type of the property to be an actual UUID object and not a `string`. What are the benefits of using a `string` here over a strongly typed UUID object?